backend/de.metas.acct.base/src/main/sql/postgresql/ddl/functions/make_unique.sql

```diff
@@ -0,0 +1,74 @@
+DROP FUNCTION IF EXISTS make_unique(varchar,
```

Comment:
@adi-stefan  cc @metas-ts 

I know that I am too late at the party,
but IMHO this function shall not be in the `de.metas.acct` project but in adempiere/base.